torch/nn/modules/module.py

```diff
@@ -531,6 +533,35 @@ def get_buffer(self, target: str) -> "Tensor":
 
         return buffer
 
+    def get_extra_state(self) -> Dict[str, Any]:
+        """
+        Returns any extra state to include in the module's state_dict.
+        Implement this and a corresponding :func:`set_extra_state` for your module
+        if you need to store extra state. This function is called when building the
+        module's `state_dict()`.
+
+        Note that extra state should be pickleable to ensure working serialization
+        of the module. We only provide provide backwards compatibility guarantees
+        for serializing Tensors; other objects may break backwards compatibility if
+        their serialized pickle form changes.
+
+        Returns:
+            dict or OrderedDict: Any extra state to store in the module's state_dict
```

Comment:
Aren't all dicts OrderedDicts now anyways?

Reply:
True; changed to just read 'dict'

Reply:
Changed again to read 'object'